src/interface/src/app/plan/plan.component.ts

```diff
@@ -29,6 +31,20 @@ export class PlanComponent implements OnInit, OnDestroy {
   planOwner$ = new Observable<User | null>();
   planNotFound: boolean = false;
   showOverview$ = new BehaviorSubject<boolean>(false);
+  // TODO this should show the scenario name if looking for configuration/scenario.
+  // for now displaying scenario id
+  breadcrumbs$ = combineLatest([
+    this.currentPlan$,
+    this.planService.planState$,
+  ]).pipe(
+    map(([plan, planState]) => {
+      const crumbs = plan ? [plan.name] : [];
+      if (planState.currentConfigId) {
```

Comment:
Would you mind actually waiting until my scenario name pr is merged and using that? Realized my changes that are coming soon are going to break this since we won't have a scenario id (configid) until after the scenario is created with the new flow

Reply:
@sachiyuan let me know if this is good to go! thanks :D

Reply:
Yes, the scenario name changes are merged!